### ISSUE TITLE
Filename overlaps with owner name when "lls" command

### ISSUE THREAD

**Issue by Zwirek009** [filiprak/fileShare#28]
![image](https://cloud.githubusercontent.com/assets/10967805/26730057/071de72c-47b0-11e7-91f1-4f338064b536.png)

**Comment by filiprak** [comment]
Yes, it need to be done, @Derdi is working on list printing format ;)
